From 9f48bed40f514352e0f97378e1fea582a0443f9f Mon Sep 17 00:00:00 2001
From: Erez_Zadok <ezk@cs.sunysb.edu>
Date: Wed, 11 Jul 2007 16:13:07 -0400
Subject: [PATCH] Unionfs: use the splice_read file method instead of sendfile

In 2.6.23 (prior to rc1), sendfile was obsoleted in favor of splice_read.

Signed-off-by: Erez Zadok <ezk@cs.sunysb.edu>
---
 fs/unionfs/file.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fs/unionfs/file.c b/fs/unionfs/file.c
index e978766..ab243cc 100644
--- a/fs/unionfs/file.c
+++ b/fs/unionfs/file.c
@@ -153,5 +153,5 @@ struct file_operations unionfs_main_fops = {
 	.flush		= unionfs_flush,
 	.release	= unionfs_file_release,
 	.fsync		= file_fsync,
-	.sendfile	= generic_file_sendfile,
+	.splice_read	= generic_file_splice_read,
 };
-- 
1.5.4.3

